pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -892,6 +892,14 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 		return nil, err
 	}
 
+	// Resize and notify the VM about changed disks
+	for _, disk := range domain.Spec.Devices.Disks {
+		if imageCanBeExpanded(disk) {
+			logger.V(1).Infof("Disk about to be resized by libvirt to size %d, imageCanBeExpanded(disk) %v", disk.Size, imageCanBeExpanded(disk))
```

Comment:
Logging the return value of `imageCanBeExpanded(disk)` seems unnecessary, but logging `disk` itself would probably be a good idea.